docs/source/optimizations/ibm/ibmq_dd.ipynb

```diff
@@ -0,0 +1,349 @@
+       "     ┌─────────┐                                           ┌─────────────┐┌─────────┐┌─────────┐┌─────────┐┌─────────────┐┌─────────┐┌─────────┐             ░ ┌─┐         \n",
+       "q_1: ┤ Rx(π/2) ├───────────────────────────────────────────┤1            ├┤ Rz(π/2) ├┤ Rx(π/2) ├┤ Rz(π/2) ├┤1            ├┤ Rz(π/2) ├┤ Rx(π/2) ├─────────────░─┤M├─────────\n",
+       "     ├─────────┤ ┌─────────┐                               │             │└─────────┘└─────────┘└─────────┘│  Acecr(π/2) │├─────────┤├─────────┤┌──────────┐ ░ └╥┘┌─┐      \n",
+       "q_2: ┤ Rz(π/2) ├─┤ Rx(π/2) ├───────────────────────────────┤  Acecr(π/2) ├─────────────────────────────────┤0            ├┤ Rz(π/2) ├┤ Rx(π/2) ├┤ Rz(3π/2) ├─░──╫─┤M├──────\n",
+       "     ├─────────┤ └─────────┘┌─────────────┐┌─────────┐┌───┐│             │                                 └─────────────┘└─────────┘└─────────┘└──────────┘ ░  ║ └╥┘┌─┐   \n",
+       "q_3: ┤ Rx(π/2) ├────────────┤1            ├┤ Rz(π/2) ├┤ X ├┤0            ├───────────────────────────────────────────────────────────────────────────────────░──╫──╫─┤M├───\n",
+       "     └─────────┘            │             │└─────────┘└───┘└─────────────┘                                                                                   ░  ║  ║ └╥┘   \n",
+       "q_4: ───────────────────────┤  Acecr(π/2) ├─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╫──╫──╫────\n",
+       "     ┌──────────┐┌─────────┐│             │                                                                                                                  ░  ║  ║  ║ ┌─┐\n",
+       "q_5: ┤ Rz(3π/2) ├┤ Rx(π/2) ├┤0            ├──────────────────────────────────────────────────────────────────────────────────────────────────────────────────░──╫──╫──╫─┤M├\n",
+       "     └──────────┘└─────────┘└─────────────┘                                                                                                                  ░  ║  ║  ║ └╥┘\n",
+       "q_6: ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╫──╫──╫──╫─\n",
+       "                                                                                                                                                                ║  ║  ║  ║ \n",
+       "c: 4/═══════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╩══╩══╩══╩═\n",
+       "                                                                                                                                                                2  3  1  0 "
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "qc_compiled.circuit.draw(fold=-1)"
```

Comment:
micro-nit: I usually add `qc.draw()` to the same cell defining/generating it, just to be more compact

Reply:
That's fair! Went ahead and made the change.